FAQs/README.md

```diff
@@ -70,9 +71,34 @@ To update the telemetry preferences, go to resource group where AzTS solution ha
   - If AIConfigurations\_\_AnonymousUsageTelemetry\_\_LogLevel is either 'All' or 'Onboarding', then specify the team's contact DL in this.
   - If AIConfigurations\_\_AnonymousUsageTelemetry\_\_LogLevel is either 'Anonymous' or 'None', then specify 'N/A' in this.
   
+### **5. On running the Autoupdater function getting some error related to function runtime version.**
+
+
+- Step-1: Run below command for AzSK-AzTS-AutoUpdater-xxxxx function app. 
+    ``` Powershell
+
+    $appSettings = @{}
+    $appSettings.add("FUNCTIONS_EXTENSION_VERSION","~4")
+    $appSettings.add("WEBSITE_RUN_FROM_PACKAGE","https://aka.ms/DevOpsKit/AzTS/V4/AutoUpdater")
+
+    # Update function runtime and package path.
+    Update-AzFunctionAppSetting -AppSetting $appSettings -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME> -Force
+
+    # Update Net framework version
+    Set-AzWebApp -NetFrameworkVersion v6.0 -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME>
+    ``` 
+- Step-2 : Run below command for AzSK-AzTS-MetadataAggregator-xxxxx and AzSK-AzTS-WorkItemProcessor-xxxxx.
+
+    ``` Powershell
+    # Update function runtime. 
+    Update-AzFunctionAppSetting -AppSetting @{"FUNCTIONS_EXTENSION_VERSION" = "~4"} -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME> -Force
+
+    # Update Net framework version
+    Set-AzWebApp -NetFrameworkVersion v6.0 -Name <APP_NAME> -ResourceGroupName <RESOURCE_GROUP_NAME>
+    ```
+- Step-3. Re-run the AutoUpdater function after completing Step 2 and the operation should complete successfully.This also ensures that the function runtime is updated to V4 for AzSK-AzTS-AutoUpdater-xxxxx, AzSK-AzTS-MetadataAggregator-xxxxx and AzSK-AzTS-WorkItemProcessor-xxxxx function app.
```

Comment:
- Step-3. Re-run the AutoUpdater function after completing Step 2 and the operation should complete successfully. This also ensures that the function runtime is updated to V4 for AzSK-AzTS-AutoUpdater-xxxxx, AzSK-AzTS-MetadataAggregator-xxxxx and AzSK-AzTS-WorkItemProcessor-xxxxx function app.